NetKAN/FusTek-SharedAssets.netkan

```diff
@@ -0,0 +1,19 @@
+{
+    "spec_version"   : "1.4",
+    "identifier"     : "FusTek-SharedAssets",
+    "$kref"          : "#/ckan/github/sumghai/FusTek_SharedAssets",
+    "name"           : "FusTek Aerospace - Shared Assets",
+    "abstract"       : "Agency folders shared by multiple FusTek-branded add-ons for Kerbal Space Program ",
+    "license"        : "unknown",
+    "release_status" : "stable",
+    "x_netkan_license_ok": true,
+    "resources" : {
+        "repository"   : "https://github.com/sumghai/FusTek_SharedAssets"
```

Comment:
Any `homepage`? It's not strictly necessary, but is a nice to have for users so they know where to direct questions, get info, etc.